scilpy/tractanalysis/reproducibility_measures.py

```diff
@@ -179,6 +179,57 @@ def approximate_surface_node(roi):
     return count
 
 
+def compute_fractal_dimension(density, n_steps=10, box_size_min=1.0,
+                              box_size_max=None, base=2.0,
+                              threshold=0.0, box_size=None):
+    """
+    Compute the fractal dimension of a bundle to measure the roughness.
+    The code is extracted from https://github.com/FBK-NILab/fractal_dimension
+    Parameters
+    ----------
+    density: ndarray
+        A ndarray where voxel values represent the density of a bundle.
+    n_steps: int
+    box_size_min: float
+        The minimum size of a box.
```

Comment:
This description does not help to understand the parameters.
What is its influence, what is a good range, how to choose it, when/why would it be better to fix it manually?
This apply to all parameters below